### PR TITLE
fix doc for PathFollower3D

### PR DESCRIPTION
**Pull request by xiongyaohua**
fix doc for "h_offset" and"v_offset" on the PathFollower3D

### CODE REVIEW

**Comment by mhilbrunner** [comment]
cc @smix8

**Comment by akien-mga** [comment]
There's relevant info on these properties in https://github.com/godotengine/godot/pull/64808#issuecomment-1225717248

CC @KoBeWi, maybe you can validate the added docs / propose something different.

File: doc/classes/PathFollow3D.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
 			There are two answers to this problem: either increase the number of cached points and increase memory consumption, or make a cubic interpolation between two points at the cost of (slightly) slower calculations.
 		</member>
 		<member name="h_offset" type="float" setter="set_h_offset" getter="get_h_offset" default="0.0">
-			The node's offset along the curve.
+			The node's offset perpendicular to the curve, to the right direction.
```
Comment (KoBeWi) [inline doc/classes/PathFollow3D.xml:19]: I checked that and the old description is correct. The `h_offset` offsets the follower along the "forward" direction.

File: doc/classes/PathFollow3D.xml [hunk 2]
```diff
@@ -31,7 +31,7 @@
 			The distance from the first vertex, considering 0.0 as the first vertex and 1.0 as the last. This is just another way of expressing the offset within the path, as the offset supplied is multiplied internally by the path's length.
 		</member>
 		<member name="v_offset" type="float" setter="set_v_offset" getter="get_v_offset" default="0.0">
-			The node's offset perpendicular to the curve.
+			The node's offset perpendicular to the curve, to the up direction.
```
Comment (KoBeWi) [inline doc/classes/PathFollow3D.xml:34]:
```suggestion
			The node's offset perpendicular to the curve, along the up direction.
```
Maybe `vector` would be better than `direction`.

**Comment by KoBeWi** [comment]
> There's relevant info on these properties in https://github.com/godotengine/godot/pull/64808#issuecomment-1225717248
CC @KoBeWi, maybe you can validate the added docs / propose something different.

3D follower seems to be the same as 2D one, which IMO doesn't make much sense. `h_offset` would be more useful if it was perpendicular to the curve. Or maybe it would even make sense to change `h_offset` to a Vector2. But that's out-of-scope of this PR, as it's about docs.

**Comment by mhilbrunner** [comment]
This needs to be salvaged. @xiongyaohua, feel free to resubmit or ping for a reopen if you are happy to make the above changes, and please don't mark conversations as resolved if they weren't resolved. Thanks for contributing!

**Pull request closed without merging**